.github/workflows/prioritize-by-reactions.yaml

```
@@ -0,0 +1,93 @@
+name: Prioritize Issues by Reactions
+
+on:
+   schedule:
+     - cron: '0 0 * * *'   # Runs once a day at midnight UTC
+
+jobs:
+  prioritize:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Install jq
+        run: sudo apt-get install -y jq
+
+      - name: Prioritize issues by reactions
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: NuGet/NuGetGallery
+          BUG_HIGH: 10
+          BUG_MEDIUM: 5
+          FEATURE_MEDIUM: 25
+        run: |
+          issue_numbers=$(gh issue list -R "$REPO" --state open --json number -q '.[].number')
+
+          for ISSUE in $issue_numbers; do
+            echo "🔍 Checking Issue #$ISSUE"
+
+            # Get issue labels
+            labels=$(gh issue view $ISSUE -R $REPO --json labels -q '.labels[].name' | tr '\n' ' ')
+            echo "   Labels: $labels"
+
+            reactions=$(curl -s \
+              -H "Accept: application/vnd.github.squirrel-girl-preview+json" \
+              -H "Authorization: token $GH_TOKEN" \
+              https://api.github.com/repos/$REPO/issues/$ISSUE/reactions)
+
+            total=$(echo "$reactions" | jq 'length')
+
+            # Determine priority based on issue type
+            if echo "$labels" | grep -q "Type:Bug"; then
+              echo "   🐛 Bug issue detected"
+              if [ "$total" -ge "$BUG_HIGH" ]; then
+                priority="Priority - 1"
+              elif [ "$total" -ge "$BUG_MEDIUM" ]; then
+                priority="Priority - 2"
+              else
+                priority="Priority - 3"
+              fi
+              echo "👉 Issue #$ISSUE has $total reactions — Assigning label: $priority"
+            elif echo "$labels" | grep -q "Type:Feature" || "$labels" | grep -q "feature-request"; then
```

```suggestion
            elif echo "$labels" | grep -q "Type:Feature" || echo "$labels" | grep -q "feature-request"; then
```

I think we need this here ^